src/info.rs

```diff
@@ -75,6 +78,7 @@ impl From<Settings> for RelayInfo {
 
         let limitations = Limitation {
             payment_required: Some(p.enabled),
+            restricted_writes: Some(p.enabled || c.verified_users.is_enabled() || c.authorization.pubkey_whitelist.is_some()),
```

Comment:
Should there also be an option in the config file in the case that the restriction is placed with a grpc plugin

Reply:
Yes actually, because this is exactly how i use it, and ive setup a hacky rewrite of the json to do this